### PR TITLE
[Test] Move Re-generate code coverage to weekly pull request

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

File: .github/workflows/weekly_pull_requests.yaml [hunk 1]
```diff
@@ -32,7 +50,7 @@ jobs:
                 uses: shivammathur/setup-php@v2
                 with:
                     php-version: 7.4
```
Comment (samsonasik) [inline .github/workflows/weekly_pull_requests.yaml:52]: @TomasVotruba should it be 7.3 same as code coverage action?

Comment (TomasVotruba) [reply .github/workflows/weekly_pull_requests.yaml:52]: I think 7.4 is ok here

**Review by TomasVotruba — APPROVED** [review]

**Comment by TomasVotruba** [comment]
Thank you

**Comment by samsonasik** [comment]
Just realized that regenerate code coverage doesn't need a pull request, it may just need create branch - run coverage - delete branch

**Pull request merged**